package.json

```diff
@@ -36,7 +36,8 @@
       "got@<11.8.5": ">=11.8.5",
       "minimatch@<3.0.5": ">=3.1.2",
       "loader-utils": ">=3.2.1",
-      "terser": ">=5.14.2"
+      "terser": ">=5.14.2",
+      "eta@<2.0.0": ">=2.0.0"
```

Comment:
Is this required? I thought you said no overrides were needed?

Reply:
Yeah there was a vulnerability in a /docs dependency actually, I removed the other ones that were fixed by an update though.